src/lightning/store/README.md

```diff
@@ -13,87 +12,30 @@
 
 ```python
 import lightning as L
-from sample.model import LitAutoEncoder, Encoder, Decoder
 
-# Initialize your model here
-autoencoder = LitAutoEncoder(Encoder(), Decoder())
+# Upload a checkpoint:
+L.store.upload_model("mnist_model", "mnist_model.ckpt")
```

Comment:
Several frameworks (e.g. transformers) chunk up the checkpoint into multiple files.
Also, several models come with a model checkpoint and a tokenizer checkpoint.

We need to enable uploading multiple files under a single checkpoint name.